packages/salesforcedx-vscode-core/src/telemetry/telemetryReporter.ts

```diff
@@ -0,0 +1,174 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+'use strict';
+
+// process.env['APPLICATION_INSIGHTS_NO_DIAGNOSTIC_CHANNEL'] = true;
```

Comment:
This seems to be there because of how `applicationinsights` uses the diagnostic channel (https://github.com/Microsoft/vscode-extension-telemetry/issues/16). Did you look into this ?

Reply:
I haven't looked into this, but I will thanks for the link

Reply:
From what I'm understanding, the original need for that line was as a workaround to ensure that there isn't any monkey patching, but as of `1.0.5` of the `applicationinsights` module this workaround is no longer needed. We're currently using `1.0.6` so I believe we shouldn't need to include this line